src/hrflow_connectors/connectors/greenhouse/actions.py

```diff
@@ -3,7 +3,7 @@
 import html
```

Comment:
Remove unnecessary import html because we already have it now in `remove_html_tags` function

Reply:
It uses here 
```python
description_html = html.unescape(description_content)
```
line 76

And it is not in `remove_html_tags` (and i don't want)